src/irmin-pack/unix/io_legacy.ml

```diff
@@ -136,15 +136,14 @@ module Unix : S = struct
             file
```

Comment:
Why do we still need `Io_legacy`? Is it only for atomic writes (used for storing branches)? Could we move this in the file manager as well? (to keep everything in one file?)

Reply:
This is tracked by https://github.com/mirage/irmin/issues/1955